test/dune

```
@@ -5,6 +5,7 @@
   alcotest
   alcotest-lwt
   ppx_deriving_yojson.runtime
+  client-utilities
```

```suggestion
  ocaml-ci-client-utilities
```
Same: can be ignored.